persistence/subsystem/offsetstore/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package offsetstore defines an API for persisting the application event
+// stream offset.
```

Comment:
```suggestion
// Package offsetstore defines an API for persisting event offsets
// representing "progress" through an application's event stream.
```

эти трудно хорошо объяснить, может быть этот текст лучше... я не уверен